### PR TITLE
Move SM header into web

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Move the SM header to the web folder layout. This opens up re-using the header for the regular web vault.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: bitwarden_license/bit-web/tsconfig.json [hunk 1]
```diff
@@ -11,5 +11,5 @@
       "@bitwarden/web-vault/*": ["../../apps/web/src/*"]
     }
   },
-  "include": ["src/**/*.stories.ts"]
+  "exclude": ["src/**/*.stories.ts"]
```
Comment (Thomas-Avery) [inline bitwarden_license/bit-web/tsconfig.json:14]: Why are we moving from an `include` to an `exclude`?

**Review by Thomas-Avery — COMMENTED** [review]
One question, everything else looks good.

**Review by Thomas-Avery — DISMISSED** [review]
Looks good, thank you!

**Review by willmartian — APPROVED** [review]
LGTM 🌈

**Review by Thomas-Avery — APPROVED** [review]

**Pull request merged**